strategies/changed/pom.xml

```diff
@@ -27,6 +27,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.assertj</groupId>
```

Comment:
when we have all the test dependencies that are used in several places defined in the parent pom, could you please to put the `arquillian-spacelift` dependency there as well?